eslint.config.js

```diff
@@ -276,7 +276,7 @@ export default tseslint.config(
         },
     },
     {
-        files: ["help-beta/src/scripts/client/**"],
+        files: ["starlight_help/src/scripts/client/**"],
```

Comment:
If this first commit passes CI and runs, we can probably merge it via a separate PR ID to help shrink this branch.

Reply:
#35646